
Fix outdated ALSA API useage ...

  - Rene Rebe <rene@rocklinux.org>

--- SDL-1.2.6/src/audio/alsa/SDL_alsa_audio.c.orig	2004-01-11 20:16:20.000000000 +0100
+++ SDL-1.2.6/src/audio/alsa/SDL_alsa_audio.c	2004-01-11 20:39:32.000000000 +0100
@@ -34,6 +34,11 @@
 #include <sys/types.h>
 #include <sys/time.h>
 
+/* until convertion to the new ALSA API ... */
+
+#define ALSA_PCM_OLD_HW_PARAMS_API
+#define ALSA_PCM_OLD_SW_PARAMS_API
+
 #include "SDL_audio.h"
 #include "SDL_error.h"
 #include "SDL_audiomem.h"
